src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -45,6 +45,8 @@ internal sealed class MetricsEventSource : EventSource
     {
         public static readonly MetricsEventSource Log = new();
 
+        private const string SharedSessionId = "SHARED";
```

Comment:
As mentioned in the `diagnostics` repo, using the previously discussed "SHARED" value to signify these special sessions